test/studies/IMSuite/vertexColor/vertexColoring-serial.chpl

```diff
@@ -0,0 +1,319 @@
+ /* This file is part of IMSuite Benchamark Suite.
+
+    This file is licensed to You under the Eclipse Public License (EPL);
+    You may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+    http://www.opensource.org/licenses/eclipse-1.0.php
+
+   (C) Copyright IMSuite 2013-present.
+ */
```

Comment:
I think we need to keep that source file under the EPL license as I believe it qualifies as a derivative work.

We're going to check with someone at Cray about having this file in our repository (apache licensed), but they are on vacation this week :frowning: . My understanding is that EPL isn't quite as 'infectious' as others, but I just want to make sure before merging anything. I'll reply here as soon as I know something.

Sorry for the continued delays, and thanks for your patience!